packages/mongodb-linting-engine/src/main/kotlin/com/mongodb/jbplugin/linting/NamespaceCheckingLinter.kt

```diff
@@ -128,7 +128,8 @@ object NamespaceCheckingLinter {
             databaseDoesNotExistParser,
             collectionDoesNotExistParser,
             databaseIsNotKnown,
-            noCollectionSpecified
+            noCollectionSpecified,
+            otherwise { emptyList() }
```

Comment:
I am curious why this is needed when we already have `.orElse` chained on :136

Reply:
It's the same issue we had in #93, it will fail the parser if some other run is a success.